_data/pubs.yml

```
@@ -137,6 +137,16 @@
     - ai
     - hac
 
+- title: "FeedQUAC: Quick Unobtrusive Agent-Generated Commentary"
+  authors: ao Long, Kendra Wannamaker, Jo Vermeulen, George Fitzmaurice, Justin Matejka
```

Is this a typo here?
```suggestion
  authors: Tao Long, Kendra Wannamaker, Jo Vermeulen, George Fitzmaurice, Justin Matejka
```